experimental/python-flask/templates/simple/.appsody-config.yaml

```diff
@@ -0,0 +1 @@
+stack: appsody/python-flask:0.2.0
```

Comment:
@henrynash please change this 0.2 so it picks up the latest minor versions

Reply:
done